_posts/2022-08-05-hacking-for-cruise-control-ui.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: post
+title: "Hacking Strimzi for Cruise Control UI"
+date: 2022-10-31
+author: kyle_liberti
+---
+
+For some time now, Strimzi has offered Kubernetes native support Cruise Control allowing users to safely configure and communicate with Cruise Control through Strimzi custom resources.
```

Comment:
I wonder if this can be written in a better way. You could for example say that
* Strimzi uses Cruise Control
* But it hides it from the user
* Users interact with CC only through the Strimzi custom resources -> through the operator

You can probably also add some reasons for it:
* Better abstraction when hiding CC as an implementation detail
* Better declarative support

Reply:
Updated the sentences based on the feedback, let me know what you think!